Portal_Perfomance_Employes/Controllers/EmployeeController.cs

```diff
@@ -31,7 +32,7 @@ public async Task<IActionResult> GetEmployee(int id)
                 .FirstOrDefaultAsync(e => e.Id == id);
             return emp == null ? NotFound("Employee with specified ID was not found") : Ok(emp);
         }
-        
+
         [HttpPost]
         [Authorize(Roles = "Admin")]

```

Comment:
If we annotate the whole controller as requiring authorize, then we don't need the annotations for individual methods

Reply:
Yep, In a new commit I moved function GetEmployee from that controller to UserInfoController and make this controller only for admins